e2e/helper_test.go

```diff
@@ -123,9 +123,14 @@ func runWithDindSwarmAndRegistry(t *testing.T, todo func(dindSwarmAndRegistryInf
 	todo(runner)
 }
 
-func build(t *testing.T, cmd icmd.Cmd, dockerCli dockerCliCommand, ref, path string) {
-	cmd.Command = dockerCli.Command("app", "build", "-t", ref, path)
+func build(t *testing.T, cmd icmd.Cmd, dockerCli dockerCliCommand, ref, path string) string {
+	iidfile := fs.NewFile(t, "iid")
```

Comment:
Why don't you use `docker app build --quiet` instead?

Reply:
works as well, but I prefer iidfile :P